crates/sui-indexer/src/handlers/checkpoint_handler.rs

```diff
@@ -96,6 +97,10 @@ where
             // generation. Ideally we will want to differentiate between a real error and
             // a checkpoint not generated yet.
             while checkpoint.is_err() {
+                error!(
+                    "Failed to download checkpoint data with error {:?}, retrying...",
```

Comment:
nit: we might still want to log `next_cursor_sequence_number` since `checkpoint` might not contain the id when there's an error